src/DatePicker/DatePickerInput.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+
+import Box from '../Box';
+import DatePicker from './DatePicker';
+import DatePickerNavbar from './DatePickerNavbar';
+import Input from '../Input';
+import PopoverContent from '../Popover/PopoverContent';
+
+const CustomOverlay = ({ classNames, selectedDay, children, ...props }) => (
```

Comment:
Should CustomOverlay be in it's own file? We've been doing 1 component per file lately.

Reply:
I could go either way on this. I'm not opposed to putting it in its own.